chemicaltoolbox/autodock_vina/docking/docking.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="docking" name="Docking" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.1.2">autodockvina</requirement>
```

Comment:
To trigger tests you need to create a shed.yml file ... it looks like that: https://github.com/bgruening/galaxytools/blob/master/tools/mcl/.shed.yml